view.php

```diff
@@ -104,9 +104,9 @@
       . "<a href='$wwsetlink'>" . get_string("iframeNoShow-2", "wwassignment")
       ."</a><p align='center'></iframe></p>\n"
       );
-print("<iframe id='wwPage' src='$wwsetlink' frameborder='1' "
+print("<iframe id='wwPage' src='$wwsetlink' frameborder='0' "
       . "width='".$CFG->wwassignment_iframewidth."' "
-      . "height='".$CFG->wwassignment_iframeheight."'>"
+      . "height='".$CFG->wwassignment_iframeheight."' scrolling='no' >"
```

Comment:
I think so.  I'm on the road right now going to my fathers house in canton ny.  I'll look more closely when I get there.  I appreciate the help in bringing these modules up to the current best practices for moodle modules

Ttyl



Sent from my iPhone

On Mar 9, 2015, at 12:12 PM, Nelson Moller <notifications@github.com<mailto:notifications@github.com>> wrote:


In view.php<https://github.com/mgage/wwassignment/pull/1#discussion_r26049273>:

>        . "width='".$CFG->wwassignment_iframewidth."' "
> -      . "height='".$CFG->wwassignment_iframeheight."'>"
> +      . "height='".$CFG->wwassignment_iframeheight."' scrolling='no' >"


Thanks Michael for the links!
I was working on the wwassignment module... to get it properly I should do an upgrade of the table:
add field introformat and rename description into intro, if you agree with that change the mod_form should be updated to use the intro instead...
You can look at the module of your choice to see those fields (ex mod_label : https://github.com/moodle/moodle/blob/master/mod/label/db/install.xml
)
Do you agree if I do that?

-
Reply to this email directly or view it on GitHub<https://github.com/mgage/wwassignment/pull/1/files#r26049273>.